Feature render hints (#1435)
twilio/twilio-video.js: CHANGELOG.md

- Fixed a bug where Android Firefox Participants sometime failed to publish VP8 VideoTracks in a Group Room. (VIDEO-3736)

2.14.0 (In Progress)

> ====================

Reviewer: ```suggestion
====================

```

New Features
------------

**Idle Track Switch Off**

> - Idle Track Switch Off uses document visibility, track attachments, and the visibility of video elements to determine whether a RemoteVideoTrack should be switched off. A RemoteVideoTrack will be switched off when the document is no longer visible, no video elements are attached to the track, or when the video elements attached to the track are not visible.
> 
> This feature is available in Group Rooms and is enabled by default if your application specifies any Bandwidth Profile options during connect.

Reviewer: ```suggestion
- Idle Track Switch Off uses document visibility, track attachments, and the visibility of video elements to determine whether a RemoteVideoTrack should be switched off. A RemoteVideoTrack will be switched off when the document is no longer visible, no video elements are attached to the track, or when the video elements attached to the track are not visible. This feature is available in Group Rooms and is enabled by default if your application specifies any Bandwidth Profile options during connect.
```

```js
  const { connect } = require('twilio-video');

  const room = await connect(token, {
    name: "my-new-room",
    bandwidthProfile: {
      video: {
        idleTrackSwitchOff: true,

>         // Other Bandwidth Profile options...

Reviewer: ```suggestion
```

      }
    }
  });
```

> Note: This feature rely on applications using `attach` and `detach` methods of `RemoteVideoTrack`. If your application currently uses the underlying MediaStreamTrack to associate Tracks to video elements, you will need to update your application to use the attach/detach methods. This feature can be disabled by setting `idleTrackSwitchOff` property to false in the BandwidthProfileOptions dictionary.

Reviewer: ```suggestion
Note: This feature relies on applications using the `attach` and `detach` methods of a RemoteVideoTrack. If your application currently uses the underlying MediaStreamTrack to associate RemoteVideoTracks to video elements, you will need to update your application to use those methods. This feature can be disabled by setting `idleTrackSwitchOff` property to false in the VideoBandwidthProfileOptions dictionary.
```

```js
  const { connect } = require('twilio-video');

  const room = await connect(token, {
    name: "my-new-room",
    bandwidthProfile: {
      video: {
        idleTrackSwitchOff: false,

>         // Other Bandwidth Profile options

Reviewer: ```suggestion
```

  });

  ```

  You can also toggle a blur filter on a RemoteVideoTrack as shown below.

>   ```ts

Reviewer: ```suggestion
  ```js
```
